test/multiverse/suites/json/Envfile

```diff
@@ -1,15 +1,4 @@
-json_gems = ['~> 2.3.0', '~> 2.4.0', '~> 2.5.0']
-
-if RUBY_VERSION < '2.7.0'
-  json_gems += ['~> 2.2.0', '~> 1.8.6']
-end
-
-json_gems.each do |version|
-  gemfile <<-RB
-    gem 'rack'
-    gem 'json', '#{version}'
-    #{ruby3_gem_webrick}
-  RB
-end
-
-# vim: ft=ruby
+gemfile <<-RB
```

Comment:
I'm curious, why was this Envfile refactored to remove all JSON gem versions?

Reply:
I think the consensus was that Ruby ships with JSON after a certain version so isn't necessary to test. @fallwith brought this knowledge in—there might be more to this.

This started because SimpleCov has a JSON dependency that conflicted when we tested JSON. Only testing on Ruby >= 2.7.0 was part of that change, and then recognizing that maybe it was no longer needed to test JSON versions.

Reply:
Yep. We no longer support older Rubies that don't ship with the `json` gem. And by having Bundler manage `json` instead of simply relying on the system instance of the gem existing causes version compatibility conflicts with the version that Simplecov needs (the ol' "already activated version x" error). This change seems to address everything with no loss in testing functionality.

Reply:
Thanks for explaining this, @hannahramadan and @fallwith!